api/src/main/java/org/pmiops/workbench/audit/adapters/WorkspaceAuditAdapterService.java

```diff
@@ -0,0 +1,9 @@
+package org.pmiops.workbench.audit.adapters;
+
+import org.pmiops.workbench.model.Workspace;
+
+public interface WorkspaceAuditAdapterService {
+  void fireCreateAction(Workspace createdWorkspace, long dbWorkspaceId);
+
+  void fireDeleteAction(org.pmiops.workbench.db.model.Workspace dbWorkspace);
```

Comment:
I'd love to ideally. The problem is, the api request doesn't have the long ID (which I need), and the delete call doesn't use the API model. I don't trust the mapping utilities either, so this seemed like the least intrusive way to go. I can revisit that when I do the update instrumentation. 

Since I don't want to depend on the `WorkspacesController` (as it depends on me), any improvements in signature consistency will come at the cost of extra grungy code.

Reply:
I'll revisit this on the next go-round. The main goal of this interface design is minimal intrusion into the controller code.